src/app/components/main-window/sidebar/sidebar.component.html

```diff
@@ -214,6 +214,11 @@
                       Template Addon
                     </a>
                   </li>
+                  <li>
+                    <a href="#" routerLinkActive="active" [routerLink]="'gameobject/gameobject-questitem'">
+                      Quest Items
+                    </a>
+                  </li>
```

Comment:
all editors except the main one (in this case `gameobject_template`) should be moved in the `*ngIf="!gameobjectHandlerService.isNew"` below